analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ForLoopConditionAlwaysFalse.CSharp11.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace Tests.Diagnostics
+{
+    class Program
+    {
+        void LoopTest(int x, int y)
+        {
+            for (IntPtr i = 9; i >= 5;) { } // Compliant
+            for (UIntPtr i = 9; i >= 5;) { } // Compliant
+
+            for (IntPtr a = 0, b = 1; b < 1;) { } // Noncompliant
+            for (UIntPtr a = 0, b = 1; b < 1;) { } // Noncompliant
```

Comment:
I would remove `a = 0,` from these two cases. It doesn't look relevant. Do I miss something?

Reply:
It has the same conditions for the C# 9 testcase, should i remove it from there as well?
Snippet:

```csharp
            for (nint a = 0, b = 1; b < 1;) { } // Noncompliant
            for (nuint a = 0, b = 1; b < 1;) { } // Noncompliant
```

Reply:
Yes, please